deploy/kubernetes/base/controller/max-grpc-log-length.yaml

```diff
@@ -0,0 +1,4 @@
+# Set max-grpc-log-length for attacher sidecar.
+- op: add
+  path: /spec/template/spec/containers/0/args/-
```

Comment:
csi-attacher is the 2nd container: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/cf727bbf6c79ee91aa77c27220f2b5babc00d1c1/deploy/kubernetes/base/controller/controller.yaml

Reply:
Updated, and the locally build is:
```
- args:
        - --v=5
        - --csi-address=/csi/csi.sock
        - --http-endpoint=:22012
        - --leader-election
        - --leader-election-namespace=$(PDCSI_NAMESPACE)
        - --timeout=250s
        - --max-grpc-log-length=10000
        env:
        - name: PDCSI_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
        image: k8s.gcr.io/sig-storage/csi-attacher:v4.2.0
        livenessProbe:
          failureThreshold: 1
          httpGet:
            path: /healthz/leader-election
            port: http-endpoint
          initialDelaySeconds: 10
          periodSeconds: 20
          timeoutSeconds: 10
        name: csi-attacher
        ports:
        - containerPort: 22012
          name: http-endpoint
          protocol: TCP
        volumeMounts:
        - mountPath: /csi
          name: socket-di
```